CodeFormatter/formatoptions.h

```diff
@@ -112,6 +112,16 @@ enum PHPFixserFormatterStyle {
     kPcfConcatSpaceOne = (1 << 19),
     kPcfEmptyReturnStrip = (1 << 20),
     kPcfEmptyReturnKeep = (1 << 21),
+    kPcfCombineConsecutiveUnsets = (1 << 22),
+    kPcfLinebreakAfterOpeningTag = (1 << 23),
+    kPcfMbStrFunctions = (1 << 24),
+    kPcfNoBlankLinesBeforeNamespace = (1 << 25),
+    kPcfNoMultilineWhitespaceBeforeSemicolons = (1 << 26),
+    kPcfNoNullPropertyInitialization = (1 << 27),
+    kPcfNoPhp4Constructor = (1 << 28),
+    kPcfNoShortEchoTag = (1 << 29),
+    kPcfNoUnreachableDefaultArgumentValue = (1 << 30),
+    kPcfNoUselessElse = (1 << 31),
```

Comment:
I'm running in to an issue here. enum has a limit of 32 items, but i will probably need about 250 to support all options. One thing I could do is to only split it up in to an enum per widget, but even like that i think there is a chance that i will have 60 options in one widget. Would you suggest just splitting it up in multiple widgets as well or is there another way to handle this?